tests/functional/modules/test_zos_copy_func.py

```diff
@@ -1017,13 +1017,13 @@ def test_copy_non_existent_file_fails(ansible_zos_module, is_remote):
 
 @pytest.mark.uss
 @pytest.mark.parametrize("src", [
-    dict(src="/etc/profile", is_file=False, is_binary=False, is_remote=False),
-    dict(src="/etc/profile", is_file=False, is_binary=False, is_remote=True),])
-def test_copy_dest_directory_remain(ansible_zos_module, src):
+    dict(src="/etc/profile", is_remote=False),
+    dict(src="/etc/profile", is_remote=True),])
+def test_ensure_copy_file_dont_change_permission_on_dest(ansible_zos_module, src):
```

Comment:
This is a better test name, although it should read `test_ensure_copy_file_does_not_change_permission_on_dest`